biorxiv/corpora_comparison/figure_generator.py

```diff
@@ -356,7 +356,7 @@
     .append(
         full_plot_df
         .sort_values("odds_ratio", ascending=False)
-        .iloc[:-2]
+        .iloc[:-3]
```

Comment:
Is the `iloc` call here to manually get rid of infinite or NaN odds ratios? If so, would it make more sense to do it explicitly so you aren't manually hiding rows?

Reply:
>Is the iloc call here to manually get rid of infinite or NaN odds ratios? If so, would it make more sense to do it explicitly so you aren't manually hiding rows?

Actually no. The reason I have this line here is to remove non-informative tokens. For example the true leading token in this dataframe is 'u\2009'. This is a unicode for thin space character.  Not informative at all, so it needed to be removed.

Reply:
Ahhh I see